src/modules/configurator/components/ConfigurationView.tsx

```diff
@@ -0,0 +1,128 @@
+import { useEffect } from "react";
+import { CarImageSlide } from "./CarImageSlide";
+import { useRecoilValue, useRecoilState } from "recoil";
+import { userConfigurationState, configurationPriceState } from "../state";
+import { userState } from "../../authentification/state";
+import { ConfigPropertyView } from "./ConfigPropertyView";
+import { PageLoading } from "../../global/components";
+import { ConfigNavbar } from "./ConfigNavbar";
+import { useSearchParams } from "react-router-dom";
+import { fetchUserConfiguration } from "../services";
+
+export function ConfigurationView() {
+  const [configuration, setConfiguration] = useRecoilState(
+    userConfigurationState
+  );
+  const displayPrice = useRecoilValue(configurationPriceState);
+  const user = useRecoilValue(userState);
+
+  const [searchParams] = useSearchParams();
+
+  const configId = searchParams.get("configId");
+
+  const handleGetUserConfiguration = async () => {
```

Comment:
You can put this inside an useCallback with user and configId as dependency.
Maybe you want this to trigger, but you dont have the user or configId yet.

Reply:
Or you can put inside the useEffect dependencies the user and id values, but I suggest the useCallback, because there you can see what values you depend on.

Reply:
Added useCallback with user and configId dependencies.